src/dbnode/persist/fs/files.go

```diff
@@ -51,6 +51,11 @@ const (
 
 	commitLogComponentPosition    = 2
 	indexFileSetComponentPosition = 2
+
+	// This is the default size that the standard library uses. We've
+	// redefined it here to make some of public APIs nicer and not
+	// require the user to specify this value.
+	defaultBufioReaderSize = 4096
```

Comment:
You could probably make this a var and infer it yeah?  i.e. `var defaultBufioReaderSize = bufio.NewReader(nil).Size()`, I tested this and seems to work.